stdlib/2and3/optparse.pyi

```diff
@@ -156,19 +156,29 @@ class OptionGroup(OptionContainer):
     def _create_option_list(self) -> None: ...
     def set_title(self, title: _Text) -> None: ...
 
+class Values:
+    def __init__(self, defaults: Optional[Mapping[str, Any]] = ...) -> None: ...
+    def _update(self, dict: Dict[_Text, Any], mode: Any) -> None: ...
```

Comment:
These three appear to taking any Mapping, not just Dict. Why are they here at all, though? They're private methods.

Reply:
I guess this class was just moved, so it's OK to keep this as is. I'm going to apply the changes myself and merge this.